tests/unit/array/sort-test.js

```diff
@@ -0,0 +1,83 @@
+import { sort } from 'ember-awesome-macros/array';
```

Comment:
The line I'm using is unit tests use mocking (I verify `.sort()` was called on my mock) and the integration tests use the real implementation more along the lines of a smoke test.